src/operator/operator_common.h

```diff
@@ -309,6 +309,34 @@ inline void ParamParser(nnvm::NodeAttrs* attrs) {
   attrs->parsed = std::move(param);
 }
 
+/*! \brief Callback class to allow for convenient development and testing */
+template<typename Type>
+class Callbacker {
```

Comment:
ok let's move this to batchnorm, or better, remove this and only test public methods so that no callback is needed.